mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,21 @@
             <action android:name="android.intent.action.MAIN" />
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
+        <intent-filter>
+            <data android:scheme="${SCHEME}" />
```

Comment:
Let's use variables without manifest placeholders

```suggestion
            <data android:scheme="@string/APP_SCHEME" />
```

Reply:
thanks i will do it